frontend/src/components/Reports/Routine.js

```diff
@@ -27,21 +28,15 @@ export const RoutineReportsMenu = {
       icon: Microscope,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=statisticsReport",
+          link: "/StatisticsReport?type=indicator&report=statisticsReport",
           label: <FormattedMessage id= "sidenav.label.statisticsreport"/>,
         },
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=indicatorHaitiLNSPAllTests",
+          link: "/SummaryOfAllTests?type=indicator&report=indicatorHaitiLNSPAllTests",
```

Comment:
i think you can use a single path for all the agregate reports since they are loaded from the same paraent component ie 
"/AggregateReport?type=indicator&report=indicatorHaitiLNSPAllTests"